### PR TITLE
Bed management UI enhancement

### LINKED ISSUE #6974: Design Enhancement for Bed Management
We have recently redesigned the Location Management. It'd be nice to have a similar design for Bed Management too.

Current:
image

Expected:
image

### PR DESCRIPTION
**Pull request by Jacobjeevan**
## Proposed Changes

- Fixes #6974 

image

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [x] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [x] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Facility/BedManagement.tsx [hunk 1]
```diff
@@ -52,11 +52,21 @@ const BedRow = (props: BedRowProps) => {
...
   );
 
+  const getDesc = () => {
+    if (description) {
+      return description.length > 100
+        ? description.slice(0, 100) + "..."
+        : description;
+    }
+    return "-";
+  };
```
Comment (bodhish) [inline src/Components/Facility/BedManagement.tsx]: Can't we use tailwind https://tailwindcss.com/docs/text-overflow#truncate

Comment (bodhish) [reply src/Components/Facility/BedManagement.tsx]: Also lets not show anything if we don't have a description.

**Review by bodhish — CHANGES_REQUESTED** [review]

**Review by bodhish — APPROVED** [review]
LGTM

**Comment by Jacobjeevan** [comment]
I'll take a look at the failing test 👍

**Comment by samholics** [comment]
tested, working fine @nihal467 
However is it possible to to display the description..now its not showing the full description.

**Comment by nihal467** [comment]
image

- [ ] use the full width of the page, on the left side, so it is aligned with the bed management heading

image

- [ ] add a space between the white padding and the button when there is no bed in the location

**Comment by Jacobjeevan** [comment]
> image
> 
>     * [ ]  use the full width of the page, on the left side, so it is aligned with the bed management heading
> 
> 
> image
> 
>     * [ ]  add a space between the white padding and the button when there is no bed in the location

Done 👍

**Comment by nihal467** [comment]
LGTM

File: src/Locale/en.json [hunk 2]
```diff
@@ -224,6 +224,7 @@
   "add_beds": "Add Bed(s)",
   "add_details_of_patient": "Add Details of Patient",
   "add_location": "Add Location",
+  "add_new_beds": "Add New Bed(s)",
```
Comment (rithviknishad) [inline src/Locale/en.json:227]: could've been improved with https://www.i18next.com/translation-function/plurals#languages-with-multiple-plurals

**Review by rithviknishad — APPROVED** [review]

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**